scripts/setup/rewrite_yaml_files.sh

```diff
@@ -0,0 +1,320 @@
+#!/bin/bash
+
+pushd $HOME/vhive/configs/knative_yamls >/dev/null
+
+rm -rf serving-core-firecracker.yaml
+rm -rf serving-crds.yaml
+rm -rf serving-default-domain.yaml
+rm -rf serving-hpa.yaml
+rm -rf serving-post-install-jobs.yaml
+rm -rf serving-storage-version-migration.yaml
+rm -rf ../metallb/metallb-ipaddresspool.yaml
+rm -rf ../setup/kube.json
+
+wget -q https://github.com/knative-extensions/net-istio/releases/download/knative-v1.9.0/net-istio.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v1.9.0/serving-core.yaml && mv serving-core.yaml serving-core-firecracker.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v1.9.0/serving-core.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v1.9.0/serving-crds.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v1.9.0/serving-default-domain.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v1.9.0/serving-hpa.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v1.9.0/serving-post-install-jobs.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v1.9.0/serving-storage-version-migration.yaml
+
+master_node_name=$(hostname)
+
+# net-istio.yaml
+
+cat net-istio.yaml |
+    yq ' 
+    (
+        select
+        (
+               .spec.selector.matchLabels.app == "net-istio-controller"
+            or .spec.selector.matchLabels.app == "net-istio-webhook"
+        )
+        | .spec.template.spec.containers[0].resources.limits.cpu
+    ) = 2' |
+    yq ' 
+    (
+        select
+        (
+               .spec.selector.matchLabels.app == "net-istio-controller"
+            or .spec.selector.matchLabels.app == "net-istio-webhook"
+        )
+        | .spec.template.spec.containers[0].resources.limits.memory
+    ) = "10Gi"' |
```

Comment:
please define and set all the constants used in this file in the header part of the file and leave comments on why these constants are appropriate (even simple: "we set these limits high enough but to fit in the budget of a typical master node server" can do; please check with @leokondrashov )

Include CPU count, memory, etc